### ISSUE TITLE
Far Cry 5 now has flickering trees

### ISSUE THREAD

**Issue by SR-dude** [daniel-schuermann/mesa#144]
Far Cry 5 now has flickering trees. The game was fine about 36 hours ago. In the meantime,
only three source files were changed when fetching, that could triggering this for me.

-  src/amd/compiler/aco_instruction_selection.cpp
-  src/amd/compiler/aco_instruction_selection_setup.cpp
-  src/amd/compiler/aco_opcodes.py

**Comment by daniel-schuermann** [comment]
Would you be able to provide a renderdoc capture so that we can bisect the issue?

**Comment by SR-dude** [comment]
The problem also occurs with llvm. Also, I'm unable to find a good commit within the mesa code base.
Roughly two days ago, I had actually finished the game and today, decided to try and finished some side quests.  In my first playthrough, I only saw minor glitching. I can't explain why it is suddenly a problem. But it isn't ACO and I'll have to take my business elsewhere. 

In the screenshot below, the texture for the road flickers in and out. Also,  in the shot there are popping trees that disappear and reappear.
![20190926165121_1](https://user-images.githubusercontent.com/935631/65726220-91eb5480-e082-11e9-8534-a0b542a0a760.jpg)
